bridgeql/django/settings.py

```diff
@@ -7,12 +7,13 @@
 
 from bridgeql.django.exceptions import InvalidBridgeQLSettings, InvalidAppOrModelName, InvalidModelFieldName
 from bridgeql.utils import load_function
+from .helpers import get_allowed_apps
 import os
 
 
 DEFAULTS = {
     'BRIDGEQL_RESTRICTED_MODELS': {},
-    'BRIDGEQL_AUTHENTICATION_DECORATOR': ''
+    'BRIDGEQL_AUTHENTICATION_DECORATOR': '',
```

Comment:
Inorder to keep track of all the settings that bridgeql has to offer along with their default values we make use of this `DEFAULTS` dictionary. We need to add `BRIDGEQL_ALLOWED_APPS` here and provide its default value in case its not present in user's settings.
So that bridgeql settings can be accessed by `bridgeql_settings.BRIDGEQL_ALLOWED_APPS` if the setting is present then it will return the value, else it will return default value as defined here.

That way we don't need to check for `hasattr` in settings.